src/services/auroraApiClients/dnsClient/query.ts

```diff
@@ -0,0 +1,37 @@
+import gql from 'graphql-tag';
+
+export interface CnameInfosQuery {
+  cnameInfo?: CnameInfo[];
+}
+
+export interface CnameInfo {
+  status: string;
+  clusterId: string;
+  appName: string;
+  namespace: string;
+  routeName: string;
+  message: string;
+  entry: {
+    cname: string;
+    host: string;
+    ttl: number;
+  };
+}
+
+export const CNAME_INFO_QUERY = gql`
+  query getCnameInfo($affiliation: String!) {
+    cnameInfo(affiliations: [$affiliation]) {
+      appName
+      clusterId
+      message
+      namespace
+      routeName
```

Comment:
Isteden for appName og namespace så skulle det vært:
```
applicationDeployment {
  name
  namespace {
    name
  }
}
```
Kan lage en oppgave på å rette opp i det.